books/free-programming-books-en.md

```diff
@@ -1,10 +1,15 @@
 ### Index
 
 * [All](#all)
-
+* [Swift](#swift)
 
 ### All
 
 * [English, By Programming Language](free-programming-books-langs.md)
 * [English, By Subject](free-programming-books-subjects.md)
   (The list of books in English is here for historical reasons.)
+
+
+### Swift
```

Comment:
Please move this change to the english books file free-programming-books-langs.md